test/python/qobj/test_qobj_identifiers.py

```diff
@@ -35,19 +35,7 @@ def setUp(self):
         self.cr_name = cr.name
         self.circuits = [qc]
 
-    def test_builtin_qasm_simulator_py(self):
-        qobj = assemble(self.circuits)
-        exp = qobj.experiments[0]
-        self.assertIn(self.qr_name, (x[0] for x in exp.header.qubit_labels))
-        self.assertIn(self.cr_name, (x[0] for x in exp.header.clbit_labels))
-
-    def test_builtin_qasm_simulator(self):
-        qobj = assemble(self.circuits)
-        exp = qobj.experiments[0]
-        self.assertIn(self.qr_name, (x[0] for x in exp.header.qubit_labels))
-        self.assertIn(self.cr_name, (x[0] for x in exp.header.clbit_labels))
-
-    def test_builtin_unitary_simulator_py(self):
+    def test_builtin_basic_simulator(self):
```

Comment:
I think  I'm missing something obvious, but how does this test relate to the `BasicSimulator`?

Reply:
`QasmSimulatorPy` has been renamed to `BasicSimulator` :)